microprofile/microprofile-rest-client-1.4/src/main/java/org/wildfly/swarm/ts/microprofile/rest/client/v14/clientheaders/ClientResource.java

```diff
@@ -0,0 +1,21 @@
+package org.wildfly.swarm.ts.microprofile.rest.client.v14.clientheaders;
+
+import java.net.URL;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+
+@Path("/")
```

Comment:
Useless, please remove. Or use `@Path("/client")` here and remove it from the method.